ppl/src/test/java/org/opensearch/sql/ppl/utils/PPLQueryDataAnonymizerTest.java

```diff
@@ -511,6 +511,37 @@ public void testPatterns() {
                 + " variable_count_threshold=5"));
   }
 
+  @Test
+  public void testRexCommand() {
+    assertEquals(
+        "source=t | rex field=message \"(?<user>[A-Z]+)\"",
+        anonymize("source=t | rex field=message \"(?<user>[A-Z]+)\""));
```

Comment:
expect field=message is normalize to field=*

Reply:
For this one, I was actually referencing to the other regular expression related commands:

https://github.com/opensearch-project/sql/blob/994bdd5588ab5fa5b2bfd897ace786fc0c046d14/ppl/src/test/java/org/opensearch/sql/ppl/utils/PPLQueryDataAnonymizerTest.java#L470-L488

So it seems like for both parse and grok, they did not anonymize the field. So should we do that? Or keep aligning with them?